dissect/target/tools/query.py

```diff
@@ -237,7 +211,7 @@ def main():
         first_seen_output_type = default_output_type
         cli_params_unparsed = rest
 
-        for func_def in find_plugin_functions(target, args.function, False):
+        for func_def, _ in find_plugin_functions(target, args.function, False):
```

Comment:
```suggestion
        func_defs, _ = find_plugin_functions(target, args.function, False)

        for func_def in func_defs:
```
Otherwise we get a "too many values to unpack" error...